### PR TITLE
Update ru_RU.lang

### PR DESCRIPTION
**Pull request by Alexsolur**
I've tried to update translation.

### CODE REVIEW

File: src/main/resources/assets/projecte/lang/ru_RU.lang [hunk 1]
```diff
@@ -355,3 +358,51 @@ pe.zero.pedestal2=Замораживает окрестности
 pe.zero.pedestal3=Активируется каждые %s
 
 pe.debug.metainvalid.name=Неверные метаданные
+
+pe.nei.worldtransmute=World Transmutation
+pe.nei.philo=Philosopher's Stone Smelting
+
+pe.manual.index=Index
+pe.manual.index_button=ToC
+
```
Comment (williewillus) [inline src/main/resources/assets/projecte/lang/ru_RU.lang]: if the lines are not translated do not include them in the file

File: src/main/resources/assets/projecte/lang/ru_RU.lang [hunk 2]
```diff
@@ -355,3 +358,51 @@ pe.zero.pedestal2=Замораживает окрестности
...
+pe.manual.introduction.header=Введение
+
+# ImagePage headers
+pe.manual.img_alchchest.header=alchchest.png
```
Comment (williewillus) [inline src/main/resources/assets/projecte/lang/ru_RU.lang]: you can remove this too, it's just a placeholder

Comment (Alexsolur) [inline src/main/resources/assets/projecte/lang/ru_RU.lang]: I should remove last 4 strings?

**Comment by sinkillerj** [comment]
Right now its best to ignore the guide sections completely until the text is finalized.

File: src/main/resources/assets/projecte/lang/ru_RU.lang (hunk 2 above)
Comment (williewillus) [inline src/main/resources/assets/projecte/lang/ru_RU.lang]: I meant the =alchchest.png line

**Pull request merged**